service/api.go

```diff
@@ -346,6 +345,30 @@ func (as *APIServer) FetchExpansionsForTask(w http.ResponseWriter, r *http.Reque
 		as.LoggedError(w, r, http.StatusNotFound, errors.New("version not found"))
 		return
 	}
+	var u *user.DBUser
+	if t.ActivatedBy == evergreen.StepbackTaskActivator {
+		u, err = user.FindOneById(v.Author)
+	} else {
+		u, err = user.FindOneById(t.ActivatedBy)
```

Comment:
Can we only do this if t.ActivatedBy isn't empty ? (to handle the legacy case)

Reply:
done